carbon-design-system/carbon
# Tooltip fixes on hover and ESC

Closes #16152

Added an `useIsomorphicEffect` and `useCallBack` to handle the ESC key when the user is hovering the `Tooltip`.

#### Changelog

**New**

- Added `onMouseEnter` in `PopoverContent` to keep the state open when hovering the content.
- Added test for the Escape key while the tooltip trigger is on hover state.

#### Testing / Reviewing

- Go to `Tooltip` default stories
- Make sure there is no regressions.
